.github/workflows/ci.yml

```diff
@@ -172,12 +242,12 @@ jobs:
       # Builds
       - name: Build Packages
         run: rush build -t @azure/communication-react
-      # Type check test folder
-      # - must be run after building all packlets because it depends on compiled packlet dependencies
-      # - must be run only on beta flavor build because the browser test code is not conditional compiled
-      - name: Type check test folder
-        if: ${{ matrix.flavor == 'beta' }}
-        run: cd packages/react-composites && rushx tsc:e2e
+      # # Type check test folder
```

Comment:
This can probably be deleted.
The job is not run within a matrix context, so the test here is always false and does not get executed. Need to confirm this was the intention